tests/unit/woopay/test-class-woopay-session.php

```diff
@@ -0,0 +1,97 @@
+<?php
```

Comment:
This got removed in #6061.

I guess because some tests were failing with PHP 8.1 and `@runInSeparateProcess` as noted [here](https://github.com/Automattic/woocommerce-payments-private/pull/34#issuecomment-1521212197), but now without the need for the constant `REST_REQUEST` it should pass.